charts/dns/values.yaml

```diff
@@ -1,26 +1,103 @@
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  create: true
+  additionalPermissions: []
+  # - apiGroups: [""]
+  #   resources: ["nodes"]
+  #   verbs: ["list","watch"]
+  # - apiGroups: [""]
+  #   resources: ["pods"]
+  #   verbs: ["get","watch","list"]
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+service:
+  # Service that points to the http port of pdns http API.
+  # It is separated from the other services so the user can select service type freely from the others needed.
+  http:
+    enabled: true
+    type: ClusterIP
+    port: 80
+    annotations: {}
+
+  # Service that points to the UDP port of pdns.
+  # It is separated from TCP DNS port because of limitations of Kubernetes.
+  # You cannot create a service with type LoadBalancer that exposes UDP and TCP ports at the same time.
+  # Two services should workaround this limitation.
+  udp:
+    enabled: true
+    type: ClusterIP
+    port: 53
+    annotations: {}
+
+  # Service that points to the TCP port of pdns.
+  # It is separated from UDP DNS port because of limitations of Kubernetes.
+  # You cannot create a service with type LoadBalancer that exposes UDP and TCP ports at the same time.
+  # Two services should workaround this limitation.
+  tcp:
+    enabled: true
+    type: ClusterIP
+    port: 53
+    annotations: {}
+
+  # Service that points to the metrics port of externaldns.
+  externalDNSMetrics:
+    enabled: true
+    type: ClusterIP
+    port: 80
+    annotations: {}
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
 externalDNS:
+  nameOverride: ""
+  fullnameOverride: ""
+
   registry: registry.k8s.io
   repository: external-dns/external-dns
-  tag: v0.10.1
+  tag: v0.13.1
   pullPolicy: IfNotPresent
 
-  # Allows to override the name of the objects related to external-dns
-  nameOverride: ""
-
-  # If external-dns try to read istio objects without istio installed it will
-  # fail with an error "failed to sync cache: timed out waiting for the condition"
-  # This flag allows to enable or disable looking for istio's objects
-  sourceIstioObjects: false
-
-  extraConfig:
-    EXTERNAL_DNS_INTERVAL: 30s
-    EXTERNAL_DNS_EVENTS: "1"
-  # EXTERNAL_DNS_METRICS_ADDRESS: ":7979"
-
-  # EXTERNAL_DNS_REGISTRY: txt
-  # EXTERNAL_DNS_TXT_OWNER_ID: kang
-  # EXTERNAL_DNS_TXT_PREFIX: edns-managed.
-  # EXTERNAL_DNS_TXT_WILDCARD_REPLACEMENT: wildcard
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 65534
```

Comment:
Does this come from the docker image? If so, let's document it with a comment.

Reply:
It is not mandatory by the docker image. Nothing of that security context is mandatory. user `65534` is a Linux idiom:
```
❯ id nobody
uid=65534(nobody) gid=65534(nogroup) groups=65534(nogroup)
```
Some distros name `nogroup` as `nobody` too but nodoby had "max unsigned int of 16 bits" as and idiom of "the last user possible". Legacy things :)

Reply:
```suggestion
    runAsUser: 65534  # nobody
```

Reply:
Done.